.github/scripts/run-nodocker.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
```

Comment:
what is this script for? 
please answer not in github review but in comment on the header of the file :)